packages/manager/apps/telecom/src/app/telecom/pack/migration/pack-migration.less

```diff
@@ -76,4 +77,96 @@
   .subservice-title {
     font-size: 1.5em !important;
   }
+
+  .card {
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    min-width: 0;
+    word-wrap: break-word;
+    background-color: #fff;
+    background-clip: border-box;
+    border: 1px solid rgba(0, 0, 0, 0.125);
+    border-radius: 0.25rem;
+    margin: 15px;
+
+    .meeting {
+      height: 250px;
+    }
+  }
+
+  .card-body {
+    flex: 1 1 auto;
+    padding: 1.25rem;
+  }
+
+  .card-title {
+    margin-bottom: 0.75rem;
```

Comment:
**nitpick:** Avoid using custom css. Please only use the utility classes available from Bootstrap.

Reply:
I'm OK with what you said but if I did this, I didn't have the border for the card, the place of the text was not in a correct place, the look was very well. So I set custom css to have look more useable for customers.

Reply:
Can you please put a screenshot?

Reply:
![Capture d’écran 2023-03-13 à 15 12 33](https://user-images.githubusercontent.com/2368951/224727259-7f4121b4-45fe-4754-a3a8-2960b50c1256.png)

Reply:
Hello @Steffy29,
the screenshot you provided in the above message should be well achieved by using `ui-kit` and `bootstrap`. So please remove the custom css and only use the bootstrap utilities in combination with `ui-kit`.